### ISSUE TITLE
Can we have the navbar/sidenav close on every click?

### ISSUE THREAD

**Issue by 0o-de-lally** [d0minikk/materialize-meteor#14]
I think there needs to be a meteor specific adaptation to the framework. Because there are no page loads, we need to programmatically close the sidebar on click. I think this should be included by default in this package. Can you include this feature?

http://materializecss.com/side-nav.html
jQuery Plugin Methods
We have methods to show and hide your sidebar you can use to programmatically control your sidebar.

  // Show sideNav
  $('.button-collapse').sideNav('show');
  // Hide sideNav
  $('.button-collapse').sideNav('hide');

**Comment by 0o-de-lally** [comment]
Hi I think the new materialize release fixes this. Can you update this package on atmosphere? Thanks!

**Comment by elmor34** [comment]
not exactly. sidenav content always visible. Button darkens the screen but no effect.
any idea?

**Comment by 0o-de-lally** [comment]
@elmor34 there is an official release of materialize on atmospherejs.com , it includes triggers to open and close the navbar.

**Comment by elmor34** [comment]
Thanks.

I have problem wit datetime picker. Not gives same render as in your site.

Regards

Gokhan AYKAN, from Istanbul

On Tue, Apr 7, 2015 at 9:38 PM, keyscores notifications@github.com wrote:

> @elmor34 https://github.com/elmor34 there is an official release of
> materialize on atmospherejs.com , it includes triggers to open and close
> the navbar.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/d0minikk/materialize-meteor/issues/14#issuecomment-90693568
> .

## 

Gokhan AYKAN

**Comment by guimelon** [comment]
this.$(".button-collapse").sideNav({closeOnClick: true})

**Comment by K1SS4** [comment]
Add this to your .css 
#sidenav-overlay {
    display: none;
}
